sanitizers/src/test/java/com/example/ExpressionLanguageInjection.java

```diff
@@ -25,6 +25,7 @@
 import javax.el.ELProcessor;
 import javax.validation.Validation;
 import javax.validation.Validator;
+import org.apache.commons.jexl2.*;
```

Comment:
Minor: We are trying to not use wildcard imports. There is an IntelliJ setting to make it stop combining the imports.